notes/ltc-transfer-penalty.vbs

```diff
@@ -94,17 +98,10 @@ EMReadScreen app_year, 2, 12, 39
 date_of_application = (app_month & "/" & app_date & "/" & app_year)
 
 'navigating to the Uncompensated Transfer Calculation in the ELIG/HC person test
-Call navigate_to_MAXIS_screen ("elig", "HC__")
-'selects MEMB 01
-EMWriteScreen "x", 8, 26
-transmit
-'selects Person test
-EMWriteScreen "x", 7, 17
-transmit
-'selects Uncompensated transfer calculation
-EMWriteScreen "x", 18, 3
-transmit
-'reading information from Uncompensated Transfer Calculation
+Call navigate_to_MAXIS_screen ("ELIG", "HC__")        'selects MEMB 01
+Call write_value_and_transmit("X", 8, 26)             'selects Person test
+Call write_value_and_transmit("X", 7, 17)             'selects Uncompensated transfer calculation
+Call write_value_and_transmit("X", 18, 3)             'reading information from Uncompensated Transfer Calculation
 EMReadScreen row_1, 71, 5, 6
```

Comment:
I think this reads the information to add to the case note? Would you add some comments to code?